pkg/render/dex_config.go

```diff
@@ -458,8 +459,8 @@ func (d *dexConfig) Connector() map[string]interface{} {
 	}
 
 	//Google specific.
-	if d.idpSecret.Data[serviceAccountSecretField] != nil || d.idpSecret.Data[adminEmailSecretField] == nil {
-		config["serviceAccountFilePath"] = serviceAccountSecretLocation
+	if d.idpSecret.Data[serviceAccountSecretField] != nil && d.idpSecret.Data[adminEmailSecretField] != nil {
```

Comment:
The comment above this says it is google specific and in the master branch it is only done for the connectorTypeGoogle. Should we include `connectorType == connectorTypeGoogle` in the `if` statement?

Reply:
We could, but it does not really matter for two reasons:
- The other connectors in dex don't read this config, so it will be ignored when the dex code parses the config.
- There is no way a user can get their hands on this config if they are not using google.
I think we can decide to leave it the way it is.